extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -674,29 +675,33 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 						break;
 					case NetworkContainerType.BLOB_CONTAINER:
 						this._blobContainerResourceGroupDropdowns.forEach((v, index) => {
-							if (this.shouldDisplayBlobDropdownError(v, [constants.RESOURCE_GROUP_NOT_FOUND])) {
+							if (this.shouldDisplayBlobDropdownError(v, blobResourceGroupErrorStrings)) {
 								errors.push(constants.INVALID_BLOB_RESOURCE_GROUP_ERROR(this.migrationStateModel._databasesForMigration[index]));
 							}
 						});
 						this._blobContainerStorageAccountDropdowns.forEach((v, index) => {
-							if (this.shouldDisplayBlobDropdownError(v, [constants.NO_STORAGE_ACCOUNT_FOUND, constants.SELECT_RESOURCE_GROUP_PROMPT])) {
+							if (this.shouldDisplayBlobDropdownError(v, blobStorageAccountErrorStrings)) {
 								errors.push(constants.INVALID_BLOB_STORAGE_ACCOUNT_ERROR(this.migrationStateModel._databasesForMigration[index]));
 							}
 						});
 						this._blobContainerDropdowns.forEach((v, index) => {
-							if (this.shouldDisplayBlobDropdownError(v, [constants.NO_BLOBCONTAINERS_FOUND, constants.SELECT_STORAGE_ACCOUNT])) {
+							if (this.shouldDisplayBlobDropdownError(v, blobContainerErrorStrings)) {
 								errors.push(constants.INVALID_BLOB_CONTAINER_ERROR(this.migrationStateModel._databasesForMigration[index]));
 							}
 						});
 
 						if (this.migrationStateModel._databaseBackup.migrationMode === MigrationMode.OFFLINE) {
 							this._blobContainerLastBackupFileDropdowns.forEach((v, index) => {
-								if (this.shouldDisplayBlobDropdownError(v, [constants.NO_BLOBFILES_FOUND, constants.SELECT_BLOB_CONTAINER])) {
+								if (this.shouldDisplayBlobDropdownError(v, blobFileErrorStrings)) {
 									errors.push(constants.INVALID_BLOB_LAST_BACKUP_FILE_ERROR(this.migrationStateModel._databasesForMigration[index]));
 								}
 							});
 						}
 
+						if (this.migrationStateModel.isSqlVmTarget && utils.isTargetSqlVm2014OrBelow(this.migrationStateModel._targetServerInstance as SqlVMServer)) {
```

Comment:
should we centralize both of these checks into 1 function, either in stateModel or some util? we make this check a few times throughout and this coupling of check won't change, so makes sense to centralize imo as its more maintainable and readable.